src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Models/ResourcePolicy.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Altinn.Platform.Authorization.Models
+{
+    /// <summary>
+    /// This model describes the actions that can be performed for a particular resource within a app policy
```

Comment:
```suggestion
    /// This model describes the actions that can be performed for a particular resource within an app policy
```

Reply:
Based on the comments we limit this class to handle the app resource, but it looks quite generic. Consider having the comments refer to a resource in general instead of an app.